proto/client/gClient.go

```diff
@@ -156,64 +145,60 @@ func (confClient *ConfigClient) GetConfigClientConn() *grpc.ClientConn {
 	return confClient.Conn
 }
 
-func (confClient *ConfigClient) subscribeToConfigPod(commChan chan *protos.NetworkSliceResponse) {
-	logger.GrpcLog.Infoln("subscribeToConfigPod")
+// ConnectToGrpcServer connects to a GRPC server with a Client ID
+// and returns a stream if connection is successful else returns nil
+func (confClient *ConfigClient) ConnectToGrpcServer() (stream protos.ConfigService_NetworkSliceSubscribeClient) {
+	logger.GrpcLog.Infoln("connectToGrpcServer")
```

Comment:
I think this should be at the debug level.
```suggestion
	logger.GrpcLog.Debugln("connectToGrpcServer")
```

Reply:
Done